frontend/src/components/assessment-creation/CreateAssessmentPage.tsx

```diff
@@ -0,0 +1,241 @@
+    control,
+  } = useForm();
+
+  type FormInputs = {
+    assessmentName: string;
+  };
+
+  const fromErrorMsg = "Please resolve all issues before publishing or saving";
+  const [validSubmit, setValidSubmit] = useState(true);
+
+  const onSubmit = (data: any, e: any) => {
+    setValidSubmit(true);
+    console.log(data, e);
+  };
+  const onError = (errs: any, e: any) => {
+    setValidSubmit(false);
+  };
+
+  const countryOptions = React.useMemo(() => countryList().getData(), []);
+  const gradeOptions = [
+    {
+      label: "K",
+      value: "k",
+    },
+    {
+      label: "Grade 1",
+      value: "1",
+    },
+    {
+      label: "Grade 2",
+      value: "2",
+    },
+    {
+      label: "Grade 3",
+      value: "3",
+    },
+    {
+      label: "Grade 4",
+      value: "4",
+    },
+    {
+      label: "Grade 5",
+      value: "5",
+    },
+    {
+      label: "Grade 6",
+      value: "6",
+    },
+    {
+      label: "Grade 7",
+      value: "7",
+    },
+    {
+      label: "Grade 8",
+      value: "8",
+    },
+  ];
+
+  return (
+    <VStack align="left" width="75%" spacing={8}>
+      <Text textStyle="eyebrow">BASIC INFORMATION</Text>
+
+      {!validSubmit && (
+        <Alert status="error">
+          <AlertIcon />
+          {fromErrorMsg}
+        </Alert>
+      )}
+
+      <FormControl isRequired isInvalid={Boolean(errors.assessmentName)}>
+        <FormLabel color="grey.400"> Assessment Name </FormLabel>
+        <Input
+          placeholder="e.g. Ontario Grade 5 Pre-Term Assessment"
+          {...register("assessmentName", {
+            required: "Please enter a name for the assessment",
+          })}
+        />
+        <FormErrorMessage> {errors.assessmentName?.message} </FormErrorMessage>
+      </FormControl>
+
+      <Box width="50%">
+        <Controller
+          control={control}
+          name="grade"
+          rules={{ required: "Please select a grade" }}
+          render={({
+            field: { onChange, onBlur, value, name, ref },
+            fieldState: { error },
+          }) => (
+            <FormControl isRequired isInvalid={Boolean(error)}>
+              <FormLabel color="grey.400"> Grade Level </FormLabel>
+              <Select
+                name={name}
+                ref={ref}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value}
+                options={gradeOptions}
+                placeholder=""
+                chakraStyles={{
+                  dropdownIndicator: (provided) => ({
+                    ...provided,
+                    bg: "transparent",
+                    px: 2,
+                    cursor: "inherit",
+                  }),
+                  indicatorSeparator: (provided) => ({
+                    ...provided,
+                    display: "none",
+                  }),
+                  placeholder: (provided) => ({
+                    ...provided,
+                    color: "grey.300",
```

Comment:
is it possible to move this styling into the `themes` folder to keep components a bit cleaner?

Reply:
the styling is also duplicated in `TeacherSignup/SelectFormInput.tsx` so would be great if you could refactor that as well